tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "target": "es5",                                /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
```

Comment:
Suggestion: [target the features of Node 14](https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping) which would  be ES2020; the earlier targets no longer have a supported node version